sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1459,6 +1459,29 @@ def _path_residuals(
 class LinearModelCV(MultiOutputMixin, LinearModel, ABC):
     """Base class for iterative model fitting along a regularization path."""
 
+    _parameter_constraints = {
+        "eps": [Interval(Real, 0, None, closed="neither")],
+        "n_alphas": [Interval(Integral, 0, None, closed="neither")],
+        "alphas": ["array-like", None],
```

Comment:
Please update the parameter description of `alphas` in the docstrings of these estimators to
`alphas: array-like, default=None` because they currently say ndarray instead of array-like.